cmw.go

```diff
@@ -0,0 +1,195 @@
+// Copyright 2023 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmw
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/fxamacker/cbor/v2"
+)
+
+type Serialization uint
+
+const (
+	JSONArray = Serialization(iota)
+	CBORArray
```

Comment:
I am little confused as current numbering is not what exists in the implementation.
The current spec says:
       switch b[0] {
       case 0x82:
           return cborArrayDecode(b)
       case 0x5b:
           return jsonArrayDecode(b)

Reply:
I told you not to look too much into the current spec.  There are pending PRs with this content waiting to be merged.